src/components/Button.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledButton = styled.a`
+  background: ${p => p.isGradient ? 'linear-gradient(92.58deg, #0DEFE1 0%, #78FF96 100%)' : p.backgroundColor};
+  color: ${p => p.textColor};  
+  height: ${p => p.height};
+  width: ${p => p.width};
+  border-radius: ${p => p.borderRadius};
+  border: none;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-decoration: none;
```

Comment:
There's an animation on the border because of the the `transition: 200ms` line in `global.js` - you can either remove it from there (prefer this tbh) or add `transition: none` to this component.